src/main/java/io/aiven/kafka/connect/http/config/HttpSinkConfig.java

```diff
@@ -47,12 +49,15 @@ public class HttpSinkConfig extends AbstractConfig {
     public static final String KAFKA_RETRY_BACKOFF_MS_CONFIG = "kafka.retry.backoff.ms";
 
     private static final String OAUTH2_ACCESS_TOKEN_URL_CONFIG = "oauth2.access.token.url";
+    private static final String OAUTH2_GRANT_TYPE_KEY_CONFIG = "oauth2.request.grant.type.property";
```

Comment:
```suggestion
    private static final String OAUTH2_GRANT_TYPE_PROPERTY_CONFIG = "oauth2.request.grant.type.property";
```

or `_PROP_` for shortness?